phpcs.xml.dist

```diff
@@ -2,6 +2,12 @@
 <ruleset name="Ups">
   <description>Baseline coding standards</description>
 
+  <!--
+  Prevent errors caused by WordPress Coding Standards not supporting PHP 8.0+.
+  See https://github.com/WordPress/WordPress-Coding-Standards/issues/2035
+  -->
+  <ini name="error_reporting" value="E_ALL &#38; ~E_DEPRECATED" />
```

Comment:
Previously, we just told developers to use a PHP version < 8.0 (e.g. https://github.com/Upstatement/pratt-wp-theme#-system-requirements). This suppresses WPCS errors for PHP 8.0+ so we don't have to remember to do that